app/controllers/posts_controller.rb

```diff
@@ -19,9 +19,11 @@ def create
     respond_to do |format|
       format.html do
         if @new_post.save
+          flash[:notice] = 'Post created successfully.'
           redirect_to "/users/#{@new_post.author.id}/posts/", notice: 'Success!'
```

Comment:
- [ ] After creating the post we are getting message `Success!`. Instead we want to see the message `Post created successfully`. For this I think you have to remove the `notice: 'Success!'` from the line 23.